server/game/cards/13.4-BtRK/AlyceGraceford.js

```diff
@@ -0,0 +1,26 @@
+const DrawCard = require('../../drawcard.js');
+
+class AlyceGraceford extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Kneel to stand character',
+            cost: ability.costs.kneelSelf(),
+            target: {
+                activePromptTitle: 'Select another The Seven character',

```

Comment:
Generally we don't do specific prompt messages like this, unless it would be confusing otherwise. Removing `activePromptTitle` will revert it to the default ("Select a character"), which should be sufficient.

Reply:
Thank you for the review, i removed the activePromptTitle